config/checkstyle_input_checks.xml

```diff
@@ -16,16 +16,7 @@
   </module>
 
   <module name="TreeWalker">
-    <!-- Check to make sure check UT input files have config and trailing comments -->
-    <module name="Regexp">
-      <property name="id" value="ConfigCommentOnTopInputs"/>
-      <property name="format"
-               value="(\* Config)"/>
```

Comment:
hmmm, lets rewrite regex instead of removing this check
regex should be - contains `\* Config` or first line is comment `\*`

Reply:
Done. Renamed issue and PR too.